include/swift/ABI/Task.h

```diff
@@ -131,6 +131,10 @@ class alignas(2 * alignof(void*)) Job :
     return Flags.getPriority();
   }
 
+  uint32_t getJobId() const {
+    return Id;
+  }
```

Comment:
I'll need to talk to @Harjas12 or @mikeash if this is useful.

Generally we'd like to be able to print `TaskID` and I guess all jobs that matter today are just Tasks so we'll call into `getTaskId` anyway, but figured for completeness we should be handling that a Job can be not a task